src/view/workflow/index.js

```diff
@@ -9,24 +9,17 @@ cydagre(cytoscape)
 export default View.extend({
   template: `
     <div class="workflow-component">
-      <div class="workflow-buttons">
-        <button class="btn btn-default" data-hook="fit">Fit</button>
-        <button class="btn btn-default" data-hook="center">Center</button>
-        <button class="btn btn-default" data-hook="redraw">Re-draw</button>
-        <button class="btn action-required" data-hook="warning-indicator" disabled>
-          <i class="fa fa-warning"></i>
-        </button>
-      </div>
       <div class="workflow-container">
-        <div class="workflow-graph-container" data-hook="graph-container"> </div>
+        <div class="workflow-graph-container" data-hook="graph-container"></div>
       </div>
     </div>
   `,
   props: {
-    warningToggle: ['boolean', false, false],
     clearBtn: ['boolean', false, false],
     cy: ['object', false],
-    graph: 'object' // Graph (graphlib) instance
+    graph: 'object', // Graph (graphlib) instance
+    start_task_id: 'string',
```

Comment:
esta prop esta definida aca pero no se usa nunca
```suggestion
    start_task_id: 'string',
```
realmente necesita ser una prop ? 
si tiene que ser una prop , entonces porque no se esta usando ?